_examples/profile/profile.go

```diff
@@ -103,6 +104,13 @@ func profile(w http.ResponseWriter, r *http.Request) {
 					prevalue,
 				}
 			},
+			"jsonMarshallIndent": func(data interface{}) string {
+				json, err := json.MarshalIndent(data, "", "\t")
+				if err != nil {
+					fmt.Println(err)
```

Comment:
if this errors you will still return the string(json). is this intentional?

Reply:
Good question. This function is used inside the template, and since it's rendered HTML, there's not an easy way to handle the errors